cmd/ceremony/cert.go

```diff
@@ -218,7 +230,10 @@ func makeTemplate(randReader io.Reader, profile *certProfile, pubKey []byte, ct
 		return nil, fmt.Errorf("unsupported signature algorithm %q", profile.SignatureAlgorithm)
 	}
 
-	subjectKeyID := sha256.Sum256(pubKey)
+	subjectKeyID, err := generateSKID(pubKey)
```

Comment:
We had a bug here where we were:
	(a) using SHA256 rather than SHA1
	(b) hashing the full SPKI rather than just the contents of the BIT STRING that holds the actual key contents

Reply:
(This broke things like openssl ocsp request generation, which would create a issuer key hash which didn't match our SKID.)